features/providers/search_opponent_organisations.feature

```diff
@@ -11,26 +11,26 @@ Background: User is on existing opponent organisation search page
   And I click 'Save and continue'
   Then I should be on a page with title "Opponent"
 
-@javascript
+@javascript @billy
 Scenario: No results returned is seen on screen when non-existant organisation search entered
-  When I search for organisation "cakes"
+  When I search for organisation "cake"
```

Comment:
Funny change. This made me laugh. Does this actually make a difference???

Reply:
😄 no, i changed it to something else and then changed it back to `cake`.  In reality anything which is not one of the stubbed search terms will return the same stubbed response (`{ success: false, data: [] }`)

Reply:
I assumed not!